sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2573,20 +2568,22 @@
       }
     },
     "TwinProperties": {
-      "description": "Represents Twin properties",
+      "description": "Represents twin's desired and reported properties. The maximum depth of twin property objects is 10.",
       "type": "object",
       "properties": {
         "desired": {
-          "description": "Used in conjunction with reported properties to synchronize device configuration or condition. Desired properties can only be set by the solution back end and can be read by the device app. The device app can also be notified in real time of changes on the desired properties.",
+          "description": "Properties that set by the solution back end and read by the device.",
           "type": "object",
           "additionalProperties": {
+            "description": "The collection of desired property key-value pairs. They keys are UTF-8 encoded, case-sensitive and up-to 1KB in length. Allowed characters exclude UNICODE control characters (segments C0 and C1), '.', '$' and space. The desired porperty values are JSON objects, up-to 4KB in length.",
             "type": "object"
           }
         },
         "reported": {
-          "description": "Used in conjunction with desired properties to synchronize device configuration or condition. Reported properties can only be set by the device app and can be read and queried by the solution back end.",
+          "description": "Properties that are set by the device, and read and queried by the solution back end.",
```

Comment:
what does this mean by solution back end?

Reply:
It means the application that uses the service client; the msdn docs refer to service client using applications as solution back-ends, and device client using applications as device apps.